File: app/src/main/java/com/dominikgaller/alpinebooking/booking/core/inport/result/MarkBookingActiveResult.java
package com.dominikgaller.alpinebooking.booking.core.inport.result;

/**
 * Result returned by the MarkBookingActive use case.
 *
 * <p>SDD: See {@code documentation/use-cases/uc06-mark-booking-active.spec.md}, section 3.
 */
public record MarkBookingActiveResult(String status) {
}

File: app/src/main/kotlin/com/example/service/booking/core/inport/result/MarkBookingActiveResult.kt
package com.example.service.booking.core.inport.result

/**
 * Output of the MarkBookingActive use case (UC06).
 *
 * SDD: see `documentation/use-cases/uc06-mark-booking-active.spec.md` § 3.
 */
data class MarkBookingActiveResult(
    val status: String,
)
